evaluator.go

```diff
@@ -645,10 +692,47 @@ func (c *evalContext) addResult(value string) {
 	c.currentResult.Add(value)
 }
 
+func (c *evalContext) addSliceToResult(slice []interface{}) {
+	if c.currentResult.Cardinality()+len(slice) >= MaxResults {
+		panic(tooManyResults{})
```

Comment:
pretty sure, panic is inappropriate here. We don't panic anywhere else max results is exceeded. (I think we just truncate?)